package.json

```diff
@@ -139,7 +139,7 @@
     "style-loader": "^0.13.0",
     "sugarss": "^0.1.6",
     "url-loader": "^0.5.6",
-    "webpack": "2.1.0-beta.22",
+    "webpack": "^2.1.0-beta.25",
```

Comment:
Running an `npm install` or `npm update` will install a new beta version with the `^` here. Do we want that? I feel we should lock in the version so that we can test the waters a little more calmly before everything implodes.